c2corg_ui/templates/base.html

```diff
@@ -55,7 +55,7 @@
     <button class="btn btn-default menu-open-close header"><span class="glyphicon glyphicon-align-justify"></span></button>
     <div class="page-header">
       <div class="quick-search">
-        <app-simple-search dataset="wr"></app-simple-search>
+        <app-simple-search dataset="wrc"></app-simple-search>
```

Comment:
wr c ? what is c? 
Usually the letters are the first letter of a document type - Waypoint, Route, Outing, C...?